fractions/microprofile/microprofile-metrics/src/main/java/org/wildfly/swarm/microprofile/metrics/MicroprofileMetricsFraction.java

```diff
@@ -17,11 +17,16 @@
 package org.wildfly.swarm.microprofile.metrics;
 
 import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.Module;
 import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
 
 /**
  * @author Heiko W. Rupp
  */
-@DeploymentModule(name = "org.wildfly.swarm.microprofile.metrics", slot = "deployment", export = true, metaInf = DeploymentModule.MetaInfDisposition.IMPORT)
+@DeploymentModule(
```

Comment:
Can you please try and remove it altogether (sorry if it is needed after all :-) )

Reply:
I see the javadoc for `Fraction` says 

> In the event a logical "fraction" requires no configuration, a subclass of this class is not required.

I will check if it works without this class.

Reply:
Sorry, I only referred to the DeploymentModule annotation

Reply:
Unfortunately, without the fraction or with the fraction but without the `@DeploymentModule` it fails.

Reply:
Thanks Michal, sorry you spent some time on it